### DISCUSSION TITLE
Set up section

### DISCUSSION THREAD

**Discussion by kslysz** [allure-framework/allure2#1794] [Questions & Support] [answered]
Is it possible to hide "Set up" section? It is generated by @BeforeAll tag, but there is nothing I want to show in the report.
Maybe something similar to this code `Allure.getLifecycle().updateStep(stepResult -> stepResult.getParameters().clear());`, I use to remove information about step parameters?

`
 @BeforeAll
    static void beforeScenario() {...}
`
![image](https://user-images.githubusercontent.com/7330643/169038136-77e6e2b9-1468-4405-907d-dc255e661412.png)

**Comment by baev** [comment, marked as answer]
You can use Allure Lifecycle Listener and remove unnecessary fixtures from the container files:

```java
class FixtureCleanup implements ContainerLifecycleListener {
   @Override
    public void beforeContainerWrite(final TestResultContainer container) {
        if (needToBeCleaned(container)) {
            container.setBefores(new ArrayList<>());
        }
    }
}
```
